src/objectTranslations/Volunteer_Job__c-nl_NL.objectTranslation

```diff
@@ -0,0 +1,513 @@
+            <translation><!-- Pacific/Auckland --></translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Pacific/Chatham</masterLabel>
+            <translation><!-- Pacific/Chatham --></translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Pacific/Enderbury</masterLabel>
+            <translation><!-- Pacific/Enderbury --></translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Pacific/Fiji</masterLabel>
+            <translation><!-- Pacific/Fiji --></translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Pacific/Guadalcanal</masterLabel>
+            <translation><!-- Pacific/Guadalcanal --></translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Pacific/Honolulu</masterLabel>
+            <translation><!-- Pacific/Honolulu --></translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Pacific/Kiritimat</masterLabel>
+            <translation><!-- Pacific/Kiritimat --></translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Pacific/Niue</masterLabel>
+            <translation><!-- Pacific/Niue --></translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Pacific/Norfolk</masterLabel>
+            <translation><!-- Pacific/Norfolk --></translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Pacific/Pago_Pago</masterLabel>
+            <translation><!-- Pacific/Pago_Pago --></translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Pacific/Tongatapu</masterLabel>
+            <translation><!-- Pacific/Tongatapu --></translation>
+        </picklistValues>
+    </fields>
+    <gender><!-- Neuter --></gender>
+    <layouts>
+        <layout>Volunteer Job Layout</layout>
+        <sections>
+            <label><!-- Custom Links --></label>
+            <section>Custom Links</section>
+        </sections>
+        <sections>
+            <label><!-- Description --></label>
+            <section>Description</section>
+        </sections>
+        <sections>
+            <label>Locatie</label>
+            <section>Location</section>
+        </sections>
+        <sections>
+            <label>Locatie Informatie</label>
+            <section>Location Information</section>
+        </sections>
+    </layouts>
+    <nameFieldLabel><!-- Volunteer Job Name --></nameFieldLabel>
+    <webLinks>
+        <label>Vind Vrijwilligers</label>
+        <name>Find_Volunteers</name>
+    </webLinks>
+    <webLinks>
+        <label>Vrijwilligersrooster</label>
+        <name>JobRoster</name>
+    </webLinks>
+    <webLinks>
+        <label>Dienst Kalender</label>
+        <name>Job_Shift_Calendar</name>
+    </webLinks>
+    <webLinks>
+        <label>(niet gebruikt)</label>
```

Comment:
@mpusto @davidhabib - I double checked this translation and this translates to not used. I'm thinking that this is specific to their org and shouldn't be in V4S. With not being familiar with Dutch hoping there isn't more org specific translations?